### DISCUSSION TITLE
Right to Left Book View

### DISCUSSION THREAD

**Discussion by RoXimn** [sumatrapdfreader/sumatrapdf#2775] [Q&A] [answered]
Is there an option to display Right-to-Left arrangement of pages for the Book View for RTL languages?

**Comment by GitHubRulesOK** [comment, marked as answer]
Yes its  Toggle Manga Mode 
Not always shown on menu (appears for Comics but `>Toggle Manga Mode` works with other books/PDF) so best add as a user cmd
```
Shortcuts [
	[
		Cmd = CmdToggleMangaMode
		Key = Shift + M
	]
]
```
![image](https://user-images.githubusercontent.com/12754913/176960264-af731fce-cdc0-45bf-8e6c-310a4764709b.png)


You can add your own Shortcuts in latest 3.6 with Icon Bar text too so a useful option is "shortcut".
```
Shortcuts [
	[
		Cmd = CmdToggleMangaMode
		Key = Shift + M
		ToolbarText = Manga
	]
]
```

**Reply by RoXimn** [reply]
Thank you very much.
